Makefile

```diff
@@ -91,6 +91,7 @@ test:
 	&& export API_TEST_ENV_FILE_PATH=$(or $(API_TEST_ENV_FILE_PATH),tests/e2e/environments/local.postman_environment.json) \
 	&& export API_TEST_URL=$(or $(API_TEST_URL),localhost:9000) \
 	&& npm run test
+	cd sandbox && npm t
```

Comment:
This change introduces a dependency on build-spec and build-sandbox make targets. So if you were to run `make clean` and then `make test` the tests would fail. 

I think we can add these deps to the test makefile target i.e. `test: build-spec build-sandbox`
and remove them from the release target `release: build-proxy`. Then you can run test independently.  

The test and release targets get called successively in azure-pipelines so the CI should continue to function as it does now